devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy

```diff
@@ -122,6 +123,15 @@ def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulat
 	}
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.sceneactivationv1.SceneActivationSet cmd) {
+	def value = cmd.sceneId % 2 ? "pushed" : "held"
+	def childId = (int)(cmd.sceneId / 2) + (cmd.sceneId % 2)
```

Comment:
can you add a comment indicating this formula is probably only for the Aeotec devices?